CHANGELOG.rst

```diff
@@ -35,6 +35,9 @@ New Features
 
 - Add support for receiving and processing SNMP traps, and forwarding them as logs to Datadog.
 
+- APM: A new trace ingestion endpoint was introduced at /v0.5/traces which supports a more compact payload format, greatly
+  improving resource usage. The spec for the new wire format can be viewed at https://github.com/DataDog/datadog-agent/blob/7.22.0/pkg/trace/api/version.go#L21-L69.
```

Comment:
Suggestion (feel free to merge without):
```suggestion
  improving resource usage. The spec for the new wire format can be viewed `here <https://github.com/DataDog/datadog-agent/blob/7.22.0/pkg/trace/api/version.go#L21-L69>`_.
```

or 

```suggestion
  improving resource usage. See the `spec for the new wire format <https://github.com/DataDog/datadog-agent/blob/7.22.0/pkg/trace/api/version.go#L21-L69>`_ for details.
```

to make the text more reader-friendly.